include/alpaka/meta/Unique.hpp

```diff
@@ -27,10 +25,10 @@ namespace alpaka
 
             template<template<typename...> class TList, typename... Ts, typename U, typename... Us>
             struct UniqueHelper<TList<Ts...>, U, Us...>
-                : std::conditional<
-                      (Disjunction<std::is_same<U, Ts>...>::value),
+                : std::conditional_t<
+                      (std::disjunction<std::is_same<U, Ts>...>::value),
                       UniqueHelper<TList<Ts...>, Us...>,
-                      UniqueHelper<TList<Ts..., U>, Us...>>::type
+                      UniqueHelper<TList<Ts..., U>, Us...>>
```

Comment:
You could also just fold here: `std::conditional_t<(std::is_same_v<U, Ts> || ...), ...>`